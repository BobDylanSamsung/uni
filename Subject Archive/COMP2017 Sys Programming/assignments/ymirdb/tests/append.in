set a 1 2 3
set b 4 5 6
set c 9 8 7
append a 0 -1 -2
append b 2
append c 2 3 4
list entries
append a b
append c 1 b 2 a 3
list entries
bye
