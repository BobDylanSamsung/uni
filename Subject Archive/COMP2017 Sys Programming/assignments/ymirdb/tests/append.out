ok

ok

ok

ok

ok

ok

c [9 8 7 2 3 4]
b [4 5 6 2]
a [1 2 3 0 -1 -2]

ok

ok

c [9 8 7 2 3 4 1 b 2 a 3]
b [4 5 6 2]
a [1 2 3 0 -1 -2 b]

bye
